sql/hive/src/main/scala/org/apache/spark/sql/hive/hiveUDFs.scala

```diff
@@ -501,7 +501,15 @@ private[hive] case class HiveGenericUDTF(
   protected lazy val inputInspectors = children.map(toInspector)
 
   @transient
-  protected lazy val outputInspector = function.initialize(inputInspectors.toArray)
+  protected lazy val outputInspector = {
+    /**
+     * Field names are not used in HiveGenericUDTF#initialize(),
+     * so they are filled with empty strings.
+     */
+    function.initialize(ObjectInspectorFactory.getStandardStructObjectInspector(
+      (0 until inputInspectors.size).map(_ => ""),
```

Comment:
I am not so sure if we really need to change the implementation here, as even in the latest Hive code, the method `public StructObjectInspector initialize(ObjectInspector[] argOIs)` is still in used. Since the empty names here is also meaningless, how about let leave it unchanged for now?